src/ApplicationInsights.ServiceFabric.Native.Shared/RequestResponseHeaders.cs

```diff
@@ -0,0 +1,56 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Microsoft.ApplicationInsights.ServiceFabric.Module
+{
+    internal class ServiceRemotingConstants
+    {
+        /// <summary>
+        /// Request-Context header.
+        /// </summary>
+        public const string RequestContextHeader = "Request-Context";
+
+        /// <summary>
+        /// Source key in the request context header that is added by an application while making http requests and retrieved by the other application when processing incoming requests.
+        /// </summary>
+        public const string RequestContextCorrelationSourceKey = "appId";
+
+        /// <summary>
+        /// Target key in the request context header that is added to the response and retrieved by the calling application when processing incoming responses.
+        /// </summary>
+        public const string RequestContextCorrelationTargetKey = "appId"; // Although the name of Source and Target key is the same - appId. Conceptually they are different and hence, we intentionally have two constants here. Makes for better reading of the code.
+
+        /// <summary>
+        /// Source-RoleName key in the request context header that is added by an application while making http requests and retrieved by the other application when processing incoming requests.
+        /// </summary>
+        public const string RequestContextSourceRoleNameKey = "roleName";
+
+        /// <summary>
+        /// Parent Id header name
+        /// </summary>
+        public const string ParentIdHeaderName = "Microsoft.ApplicationInsights.ServiceFabric.ParentId";
+
+        /// <summary>
+        /// Correlation context header name.
+        /// </summary>
+        public const string CorrelationContextHeaderName = "Microsoft.ApplicationInsights.ServiceFabric.CorrelationContext";
+
+        /// <summary>
+        /// Dependency type value for service remoting dependency.
+        /// </summary>
+        public const string ServiceRemotingTypeName = "ServiceFabricServiceRemoting";
+
+        /// <summary>
+        /// Dependency type value for service remoting calls made across AI components.
+        /// </summary>
+        public const string ServiceRemotingTypeNameTracked = "ServiceFabricServiceRemoting (tracked component)";
```

Comment:
We've discussed about the type name and the conclusion is leaving only one type "ServiceFabricServiceRemoting" is better. The second type is added mostly to keep consistent with Http.
However, at that time, we introduced this type mostly for aggregation purpose in MDM. Since we now switched to Kusto query, this is no longer needed. Actually, added this second type will make the App Map query quite complicated.